ekip/everykid/templates/get-your-pass/educator_passes.html

```diff
@@ -41,30 +41,29 @@
 
 {% block content_inner %}
   <div class="content-inner-block">
-
+    {% get_current_language as LANGUAGE_CODE %}
     <div class="download-activity">
-      <h3>Download<br/> Activity Guides</h3>
-      <p>Activity 1: <a href="{% static "activities/scholastic-one.pdf" %}" target="_blank">Exploring Federal Lands and Waters</a> (PDF). This lesson teaches students why our country protects lands and waters.</p>
-      <p>Activity 2: <a href="{% static "activities/scholastic-two.pdf" %}" target="_blank">Environmental Stewardship</a> (PDF). This lesson shows students how to take care of lands and waters.</p>
-      <p>Activity 3: <a href="{% static "activities/scholastic-three.pdf" %}" target="_blank">Our Nation’s Native Peoples</a> (PDF). This lesson teaches students about the people who lived on this land before it was called the United States.</p>
-      <p>Activity 4: <a href="{% static "activities/scholastic-four.pdf" %}" target="_blank">Citizen Science</a> (PDF). This lesson helps kids learn about the difference between weather and climate.</p>
+      <h3>{% trans "Download" %}<br/> {% trans "Activity Guides" %}</h3>
+      <p>{% trans "Activity" %} 1: <a href="{% if LANGUAGE_CODE == 'es-us' %}{% static "activities/es_us/scholastic-one.pdf" %}{% else %}{% static "activities/scholastic-one.pdf" %}{% endif %}" target="_blank">{% trans "Exploring Federal Lands and Waters" %}</a> (PDF). {% trans "This lesson teaches students why our country protects lands and waters." %}</p>
```

Comment:
In the 'URL' for the PDF, I might use the LANGUAGE_CODE parameter, instead of hard-coding 'es-us'